csrankings.csv

```diff
@@ -11812,6 +11817,7 @@ Mark J. Weal,University of Southampton,https://www.ecs.soton.ac.uk/people/mjw7,r
 Mark James Carman,Monash University,http://data.infotech.monash.edu.au/~mcarman,NOSCHOLARPAGE
 Mark James Handley,University College London,http://www0.cs.ucl.ac.uk/staff/M.Handley,XRyUF6gAAAAJ
 Mark John Batty,University of Kent,https://www.cs.kent.ac.uk/people/staff/mjb211,MnDiDMcAAAAJ
+Mark Johnson,Macquarie University,http://web.science.mq.edu.au/~mjohnson,Z_kok3sAAAAJ

```

Comment:
Again, plainly not a DBLP name. Any common name almost certainly has a disambiguation suffix like 0001.

Reply:
> Please go through all names and verify that they are in fact DBLP entries (and change them as appropriate).

Thank you for your comment. I will create another pull request and check/modify all names per their DBLP entries.